chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/CertificationBodyManager.java

```diff
@@ -114,7 +116,7 @@ public CertificationBodyDTO update(CertificationBodyDTO acb)
     }, allEntries = true)
     public CertificationBodyDTO retire(CertificationBodyDTO acb)
             throws EntityRetrievalException, JsonProcessingException, EntityCreationException, IllegalArgumentException,
-            SchedulerException, ValidationException {
+            SchedulerException, ValidationException, MessagingException {
```

Comment:
What is changing with the ACBs that needs this exception?

Reply:
When retiring an ACB, we look at all jobs and remove that ACB from existing jobs.  It uses the existing methods methods to delete and create the updated job.  Since those methods throw a MessagingException, I let it percolate up.